e2e/playwright.portalicious.config.ts

```diff
@@ -4,4 +4,8 @@ import baseConfig from './playwright.config';
 export default defineConfig({
   ...baseConfig,
   testDir: './portalicious/tests',
+  use: {
+    ...baseConfig.use,
+    baseURL: process.env.BASE_URL_PORTALICIOUS,
```

Comment:
I think you gotta add a default value for this also in `.env.example` otherwise the tests will fail in CI

Reply:
Yes it was missing from there thanks for that